org.gabmus.unifydmin.json

```diff
@@ -0,0 +1,410 @@
+{
+    "app-id": "org.gabmus.unifydmin",
+    "command": "unifydmin",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
+    "sdk": "org.gnome.Sdk",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--talk-name=org.gnome.SettingsDaemon",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--filesystem=home:ro",
+        "--filesystem=~/.ssh:ro",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "modules": [
```

Comment:
Missing cleanup to remove unused files.

Reply:
where should I put the cleanup and what should I cleanup?

Reply:
You should put the cleanup on either each module or on the whole manifest (prefer the first case ofc). The cleanup should remove the unneeded files that get installed (like headers, documentation or anything that will just make the size of bigger without any benefits). 
You can use `flatpak-builder` in order to build the app locally and check the files that will be part of the app itself and you can remove.